app/packages/vhx-style-tables/docs/code.html

```diff
@@ -90,6 +90,38 @@
         </tr>
       </tbody>
     </table>
+
+    <!-- Block -->
+    <table class="table table--block margin-bottom-large">
+      <thead>
+        <th class="text-right" style="width: 20%">ID</th>
```

Comment:
@scottusrobus Ha! Of course. I'm not used to having those classes available outside of Foundation's `.row .column` context. Will update!